table-filter.md

```diff
@@ -241,7 +241,7 @@ employees.*
 
 > **注意：**
 >
-> 在 TiDB 生态工具中，无论表库过滤如何设置，系统库总是被排除。系统库有以下六个：
+> 在 TiDB 生态工具的默认配置中，系统库总是被排除。系统库有以下六个：
```

Comment:
@3pointer PTAL. after https://github.com/pingcap/br/pull/1048 this pr, br can explicitly restore system tables.
By explicitly set `filters` conf or `-f` command line param, use can force lightning load to system tables too, though the correctness is not guaranteed since most system tables already contains data.